dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/EJBRemoteRuntime.java

```diff
@@ -90,4 +90,11 @@
      * @param oid the exported object ID from {@link #activateAsyncResult}
      */
     void deactivateAsyncResult(byte[] oid);
+
+    /**
+     * Checks if we are able to successfully get the ORB service or not
```

Comment:
Would this be more accurate?

Checks if we are able to successfully obtain the ORB and root naming context.

`@return` true if the ORB and root name context are available

Reply:
Checks if we are able to successfully obtain the ORB and CORBA name server.

@return true if the ORB and CORBA name server are available